packages/tools/pie-icons-vue/helpers/index.js

```diff
@@ -1,5 +0,0 @@
-const removeHyphenBeforeDigits = (name) => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape
```

Comment:
I would personally keep this and use it to avoid duplication of the regex across the 3 libraries. Just keeps it a bit drier.

Reply:
I've been playing around in a local branch with this – I'll raise a PR once this is merged so this is shared